app/views/pets/show.html.erb

```diff
@@ -6,6 +6,7 @@
 <p><%= "Sex: #{@pet.sex}" %></p>
 <p><%= "Adoption Status: #{@pet.adoption_status}" %></p>
 <div class="links">
+<%= button_to "Add to Favorites", "/favorites/#{@pet.id}", method: :patch %>
```

Comment:
nice action view helper! I've really gotta start doing these instead of HTML. It is so much easier to look at.